File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height, spaces, blocks;
    // promt user for pyramid height
    do
    {
        height = get_int("Enter height: ");
    }
    while (height < 1 || height > 8);
    blocks = height;
    do
    {
        spaces = height - 1;

        // print number of spaces
        for (int i = 0; i < spaces; i++)
        {
            printf(" ");
        }

        // print the number of blocks
        for (int i = spaces; i < blocks;  i++)
        {
            printf("#");
        }

        // print 2 spaces in between
        printf("  ");

        // print the same number of blocks again
        for (int i = spaces; i < blocks;  i++)
        {
            printf("#");
        }
        printf("\n");

        // go to the next line
        height--;
    }
    while (height > 0);
}

File: pset6/mario/more/mario.py
# program prints out mario pyramid (left and right side)
from cs50 import get_int

# promt user for height of pyramid between (1-8)
while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break

blocks = height
while height > 0:
    spaces = height - 1

    # print number of spaces
    for i in range(spaces):
        print(' ', end='')

    # print number of blocks
    for i in range(spaces, blocks, 1):
        print('#', end='')

    # print out the two spaces
    print('  ', end='')

    # print out same number of blocks
    for i in range(spaces, blocks, 1):
        print('#', end='')

    # go to next line of pyramid
    print()
    height -= 1